src/cart/crt_rpc.c

```diff
@@ -1627,6 +1639,17 @@ crt_handle_rpc(void *arg)
 	D_ASSERT(rpc_priv->crp_opc_info != NULL);
 	D_ASSERT(rpc_priv->crp_opc_info->coi_rpc_cb != NULL);
 
+	if (D_LOG_ENABLED(DB_NET)) {
+		uint64_t hlc = crt_hlc_get();
+
+		if (hlc > rpc_priv->crp_create_hlc) {
+			uint64_t delay = crt_hlc2msec(hlc - rpc_priv->crp_create_hlc);
```

Comment:
looks with a problem that the crp_create_hlc was set when create/init the RPC, if the RPC is queued (when #inflight > CRT_CREDIT_EP_CTX), then probably be related long time before it really sent on-wire.
Or you may define it as crp_sent_hlc and set if when really send it over network?

Reply:
This code is from crt_handle_rpc(void *arg). So, rpc_priv->crp_create_hlc will be set when incoming RPC process the headers. Then it schedule for handling and I measure this time. So, this code will work only for new incoming RPCs.